cmd/create/admin/cmd.go

```diff
@@ -72,16 +72,18 @@ func run(cmd *cobra.Command, _ []string) {
 		os.Exit(1)
 	}
 
-	// Try to find an existing cluster admin IDP
-	existingClusterAdminIDP, _ := FindExistingClusterAdminIDP(cluster, r)
-	if existingClusterAdminIDP != nil {
-		r.Reporter.Errorf("Cluster '%s' already has an admin", clusterKey)
+	adminUser, err := r.OCMClient.GetUser(cluster.ID(), "cluster-admins", ClusterAdminUsername)
```

Comment:
@marcolan018 Is it worth defining a `const` for the `cluster-admin` group? I see we have one for `ClusterAdminUsername`.

Reply:
added new const